src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpPost.java

```diff
@@ -54,14 +58,20 @@ public void ldpcvOfLdprsMustSupportPostWithoutMementoDatetimeHeader(final String
                                         "https://fcrepo.github.io/fcrepo-specification/#ldpcv-post",
                                         ps);
 
+        // create the versioned resource
+        final String newResource = createVersionedResource(uri, info).body().asString();
+        // create a memento for the current time
+        final Response versionResponse = doPost(newResource + "/fcr:versions");
+        confirmPresenceOfLinkValue(MEMENTO_LINK_HEADER, versionResponse);
+        confirmPresenceOfVersionLocationHeader(versionResponse);
```

Comment:
Per the requirement being tested, I would expect this test to also be verifying that the state of the created memento is indeed the same the current LDPRv.